tests/utils_tests/test_functional.py

```diff
@@ -47,6 +48,31 @@ def __bytes__(self):
         self.assertEqual(str(t), "Î am ā Ǩlâzz.")
         self.assertEqual(bytes(t), b"\xc3\x8e am \xc4\x81 binary \xc7\xa8l\xc3\xa2zz.")
 
+    def cached_propety_assertions(self, attr, A):
```

Comment:
I think we'd normally call this `check_cached_property` or similar

Also I'd prefer a longer class name than `A` whilst we're refactoring these tests

Reply:
We typically follow the camel casing of unittest assertions, e.g. `assertCachedPropertyWorks`. Try to split out test refactoring in a separate PR to ease review.

Reply:
currently they are in separate commits: https://github.com/django/django/pull/10033/commits/087038896c79565424fba6b279b8e8b5b433a612